src/wallet-user/IdenfyVerificationResult.tsx

```diff
@@ -8,6 +8,17 @@ import Frame from "src/common/Frame";
 import Icon from "src/common/Icon";
 import IconTextRow from "src/common/IconTextRow";
 import "./IdenfyVerificationResult.css";
+import { getBaseUrl } from "../PublicPaths";
+import { IDENFY_PATH } from "./UserRouter";
+
+type Result = 'success' | 'error' | 'unverified';
+
+const PARAM_RESULT = "result";
+const PARAM_LOC_ID = "locId";
+
+export function resumeUrl(result: Result, locId: UUID): string {
```

Comment:
This function could go to paths/router module.

Reply:
Yes indeed.